test/common/tick.js

```diff
@@ -1,5 +1,4 @@
 'use strict';
-require('../common');
```

Comment:
Review note: This was removed because this file doesn't need to load `../common` in order to function. I can make a seperate PR if needed.